src/domain/entities/Permission.ts

```diff
@@ -20,6 +20,13 @@ export class Permission extends Struct<PermissionAttrs>() {
                 : Permission.create({ read: true, write: false }),
         };
     }
+
+    static initialPermissions(): Permissions {
```

Comment:
more declarative: noPermissions ?

(it's not sure anywhere, but we can keep it, won't hurt)